lib/ansible/modules/cloud/amazon/ec2_ami_copy.py

```diff
@@ -29,25 +29,25 @@
 options:
   source_region:
     description:
-      - The source region the AMI should be copied from.
+      - the source region that AMI should be copied from
     required: true
   source_image_id:
     description:
-      - The ID of the AMI in source region that should be copied.
+      - the id of the image in source region that should be copied
     required: true
   name:
     description:
-      - The name of the new AMI to copy.
-    required: false
+      - The name of the new image to copy
+    required: true
```

Comment:
Why change the requirement? Also, you should add an explanation in the description that this was changed for version 2.3 and that in previous versions it was not requried.